src/fabric/src/fabric2_users_db.erl

```diff
@@ -89,7 +89,8 @@ save_doc(#doc{body={Body}} = Doc) ->
         Doc#doc{body={Body3}};
     {ClearPassword, "pbkdf2"} ->
         ok = validate_password(ClearPassword),
-        Iterations = list_to_integer(config:get("couch_httpd_auth", "iterations", "1000")),
+        Iterations = chttpd_util:get_chttpd_auth_config_integer(
```

Comment:
Please change the default value back to "1000"

Reply:
Default work factor was deliberately set to 10 https://github.com/apache/couchdb/commit/f726bc4dee4a4be0d5ed15f012cfbafd2a9dd7d7. We can switch it to something even higher than 1000 now as replication is using the cookie auth by default, but it probably shouldn't be part of this commit?

If users updated the config value, that value would be in their `local.ini`  or `local.d/*.ini` files, and it would still be used after this commit. 

`default.ini` file is shipped and replaced by the couchdb packages and users should expect that file to be overwritten [1]. As I understand it, for a while it was the place to store default config settings. Later on we started adding only commented defaults in there and using code default as primary default values.  But before that, in some cases, we just forgot to update the code defaults when we updated default.ini value so they got unsynchronized.


[1] https://docs.couchdb.org/en/stable/config/intro.html
```
Warning: The default.ini file may be overwritten during an upgrade or re-installation, so localised
changes should be made to the local.ini file or files within the local.d directory.
```